chipsec/logger.py

```diff
@@ -262,6 +265,16 @@ def set_autolog_file(self):
         else:
             print('Unable to autolog')
 
+    def log_csv(self, file_name, test_cases):
+        fields = ['name', 'result', 'code', 'output']
+        if '.csv' not in file_name:
+            file_name = file_name + '.csv'
+        with open(file_name, 'w') as csv_file:
+            results_csv = csv.DictWriter(csv_file, fieldnames=fields)
+            results_csv.writeheader()
+            for test_case in test_cases:
+                results_csv.writerow(test_case.get_fields())
```

Comment:
If you wanted to omit the "output" field, you could do: `results_csv.writerow(test_case.get_fields()[:3])` I'll leave it up to you to decide if you want to keep it or night. Doesn't make a big difference.

Reply:
Doesn't work with dictionaries :(